### PR TITLE
Redesign content usage table

### PR DESCRIPTION
**Pull request by Fifok**
According to #61

I redesign content usages table.

I also split ContentTypeUsageView.tsx into content usage table components.

### CODE REVIEW

**Comment by mgorzanski** [comment]
@darekhalecki suggested to add an edit on hover:
image

File: src/Forte.Optimizely.ContentUsage/Frontend/Components/Tables/PageUrlCell/PageUrlCell.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Disclose } from "optimizely-oui";
+import React from "react";
+import PageUrlLink from "./PageUrlLink/PageUrlLink";
+
+const PageUrlCell = ({pageUrls}:{pageUrls: string[]}) => {
```
Comment (mgorzanski) [inline src/Forte.Optimizely.ContentUsage/Frontend/Components/Tables/PageUrlCell/PageUrlCell.tsx:5]: Please add an interface for props

File: src/Forte.Optimizely.ContentUsage/Frontend/Views/ContentTypeUsageView.scss [hunk 2]
```diff
@@ -1,4 +1,20 @@
...
+.forte-optimizely-content-usage-page-url {
+    padding: 0 8px;
+    width: 100%;
+    &:hover {
+        background-color: var(--grey-50);
+    }
+}
+
+.oui-disclose.is-active{
+    .oui-disclose__content {
```
Comment (mgorzanski) [inline src/Forte.Optimizely.ContentUsage/Frontend/Views/ContentTypeUsageView.scss:15]: I think you missed `&` here:
```suggestion
    & .oui-disclose__content {
```

File: src/Forte.Optimizely.ContentUsage/Frontend/Views/ContentTypeUsageView.tsx [hunk 3]
```diff
@@ -369,84 +217,25 @@ const ContentTypeUsageView = () => {
...
-                                        <Table.TD colSpan={5}>{translations.noResults}</Table.TD>
-                                    </Table.TR>
-                                )}
-                            </Table.TBody>
-                        </DiscloseTable>
-                    </div>
+          <GridCell large={12}>
+            {dataLoaded ? (
+              <div className="forte-optimizely-content-usage-table-container">
+                <ContentTypeUsagesTable rows={rows} tableColumns={tableColumns} sortDirection={sortDirection.toLowerCase()} onSortChange={handleSortChange}/>
```
Comment (mgorzanski) [inline src/Forte.Optimizely.ContentUsage/Frontend/Views/ContentTypeUsageView.tsx:223]: Please add separate lines for each of the props

**Review by mgorzanski — CHANGES_REQUESTED** [review]

**Comment by Fifok** [comment]
> 
Oh! I misunderstood that comment. Now I get it. I'll add it.

**Comment by Fifok** [comment]
> @darekhalecki suggested to add an edit on hover: image

I misunderstood that comment! Now I get it and will add it.

File: src/Forte.Optimizely.ContentUsage/Frontend/Components/Tables/ContentTypeUsagesTable/ContentTypeUsageTableRow/ContentTypeUsageTableRow.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+          table: { actions },
+        },
+      },
+    } = useTranslations();
+
+    const [isUrlHovered, urlHoveredHandlers] = useHoverTrackingHandlers();
+    const actionLabel = isUrlHovered ? "View" : "Edit";
+
+    return (
+      <Table.TR class="forte-optimizely-content-usage-table-row" onRowClick={onRowClick}>
```
Comment (mgorzanski) [inline src/Forte.Optimizely.ContentUsage/Frontend/Components/Tables/ContentTypeUsagesTable/ContentTypeUsageTableRow/ContentTypeUsageTableRow.tsx:32]: should be
```suggestion
      <Table.TR className="forte-optimizely-content-usage-table-row" onRowClick={onRowClick}>
```

File: src/Forte.Optimizely.ContentUsage/Frontend/Components/Tables/ContentTypeUsagesTable/ContentTypeUsageTableRow/ContentTypeUsageTableRow.tsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        { tableColumns
+          .filter((column) => column.visible)
+          .map((column) =>
+          <Table.TD colSpan={column.columnSpanWidth}>
```
Comment (mgorzanski) [inline src/Forte.Optimizely.ContentUsage/Frontend/Components/Tables/ContentTypeUsagesTable/ContentTypeUsageTableRow/ContentTypeUsageTableRow.tsx:36]: missing `key` here

**Review by mgorzanski — CHANGES_REQUESTED** [review]

**Review by mgorzanski — APPROVED** [review]

**Pull request merged**